system_test/helpers/aest_nodes.erl

```diff
@@ -53,10 +53,22 @@
 
 %% AWK script to keep only error, critical, alert and emergency log lines with
 %% all the extra lines following the log lines.
+%% FIXME: Temporarily ignore dispatch_worker errors, remove when PT-158762778 is merged.
+%% Example of ignored lines:
+%% 2018-07-10 15:48:59.649 [error] <0.1270.0>@aec_conductor:dispatch_worker:394 Disallowing dispatch of additional create_key_block_candidate worker
+%% 2018-07-10 15:52:10.864 [error] <0.1270.0>@aec_conductor:dispatch_worker:394 Disallowing dispatch of additional micro_sleep worker
 -define(EPOCH_LOG_SCAN_AWK_SCRIPT, "
-    /^.*\\[(error|critical|alert|emergency)\\].*$/ {
+    /^.*\\[error\\].*aec_conductor:dispatch_worker.*Disallowing dispatch of additional.*$/ {
       matched = 1
-      buff = $0
+      if (buff != \"\") {
+        buff = \"\"
+      }
+    }
+    /^.*\\[(error|critical|alert|emergency)\\].*$/ {
+      if (!matched) {
+        matched = 1
```

Comment:
I see you moved this line from before the `if` to here. I guess it is because otherwise the "aec_conductor:dispatch_worker.*Disallowing dispatch of additional" string would match again hence causing `buff` to be not empty string but `$0`?

Reply:
The `if` is added because there is now two pattern that matches the same line, this is so if it matches the first one it doesn't add the line to the buffer. With the `match = 1` before the if, it was a bug, no error lines would ever be found...